src/duk_debugger.h

```diff
@@ -41,6 +42,7 @@
 #define DUK_DBG_CMD_DETACH        0x1f
 #define DUK_DBG_CMD_DUMPHEAP      0x20
 #define DUK_DBG_CMD_GETBYTECODE   0x21
+#define DUK_DBG_CMD_APPREQUEST    0x23
```

Comment:
It would have to be first then because there's never a last command unless you hit 0xFFFFFFFF - the command numbering is not strictly logical in other ways either so we'll probably just have to suffer ;)

One option would be actually to give up on numbers and move on to strings here too. I'm not actually worried about code footprint there (after all, debugger support is optional and the difference would be small) or performance (dispatching strings is slower but not really important in the debugger). Rather, it will make commands quite a bit larger which may matter for slow links and central commands like Pause, Resume, etc.

But it might still be worth it, because it would be more modular and avoid any numbering silliness.